product_firmware_version/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
please use the new README by fragment system and generate the README.rst with the [maintainer-tools](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0)